CHANGELOG.md

```diff
@@ -7,7 +7,7 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
-## [0.0.6] - 01.10.2021
+## [0.0.10] - 04.10.2021
```

Comment:
We use YYYY-MM-DD in the other java repo changelogs to avoid differences between countries. Can we do that here too plz?

Reply:
Yeah, I see it's even in the current template.